app.json

```diff
@@ -52,7 +52,7 @@
     "RAILS_SERVE_STATIC_FILES": {
       "required": true
     },
-    "ROLLBAR_ACCESS_TOKEN": {
+    "SERVER_ROLLBAR_ACCESS_TOKEN": {
       "required": true
```

Comment:
Need to also add in the `CLIENT_ROLLBAR_ACCESS_TOKEN` var here.

Reply:
Done!